swagger.yaml

```diff
@@ -4630,3 +4630,35 @@ definitions:
         type: string
         enum: [beta, production]
         description: Level of support of the version.
+
+  operator-property:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Name of the property
+      type:
+        type: string
+        enum: ['boolean', 'string', 'number']
+        description: Type of the property
+      options:
+        type: array
+        items:
+          type: string
```

Comment:
Why to use values as strings and provide type what is in the string? Do I understand it correctly?

Reply:
The type is meant to be a hint for UI-side validation and back-end will expect the value to be parsable to it. String transport type is the one that can hold any other type.